tests/legacy-e2e/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/ShareWorkspaceMemberTest.java

```diff
@@ -43,7 +43,9 @@
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
-@Test(groups = {TestGroup.MULTIUSER, TestGroup.OPENSHIFT, TestGroup.K8S})
+// Known permanent failure https://github.com/eclipse/che/issues/15822
+@Test(groups = {UNDER_REPAIR})
+// @Test(groups = {TestGroup.MULTIUSER, TestGroup.OPENSHIFT, TestGroup.K8S})
```

Comment:
We just need to add `TestGroup.UNDER_REPAIR` to existed annotation @Test.

Reply:
Fixed.